android/engine/src/main/java/org/smartregister/fhircore/engine/datastore/ProtoDataStore.kt

```diff
@@ -85,4 +94,24 @@ class ProtoDataStore @Inject constructor(@ApplicationContext val context: Contex
       )
     }
   }
+
+  val locationCoordinates =
+    context.locationCoordinatesDatastore.data.catch { exception ->
+      if (exception is IOException) {
+        Timber.tag(TAG).e(exception, "Error reading practitioner details preferences.")
```

Comment:
Timber applys correct tags to logs, it's not necessary to invoke the `tag` function
```suggestion
        Timber.e(exception, "Error reading practitioner details preferences.")
```